### DISCUSSION TITLE
blog-2024-06-21-downgrading-paper-1-21-to-1-20-6

### DISCUSSION THREAD

**Discussion by giscus** [MrPowerGamerBR/PersonalWebsiteComments#15] [Fan Comments]
# blog-2024-06-21-downgrading-paper-1-21-to-1-20-6

During a "let's improve SparklyPower to make it the best Survival server ever" hyperfixation phase, I've really wanted to update my server to 1.21. I waited eagerly for the shiny Paper experimental 1.21 builds, to update my server right when it is released. Who cares if it is "experimental", after all, what could possibly go wrong?

https://mrpowergamerbr.com/en/blog/2024-06-21-downgrading-paper-1-21-to-1-20-6/

**Comment by Azzothebest** [comment]
Love this, is there any way I can downgrade from 1.20.6 to 1.20.4?
I've tried to search for a 1.20.4 sparkly paper jar but there is none, the artifacts expired

**Reply by MrPowerGamerBR** [reply]
Yes, it will also work, but you need to check what you will lose by downgrading, one thing off the top of my head is the new 1.20.6 item attributes.

But you would need to fork Paper to remove the item version checks! The only SparklyPaper version that has that feature is the 1.20.6 one, because that’s when I first needed that, so older versions don’t have it.
